### DISCUSSION TITLE
为啥设置了目录标题的段前，段后高度后就没法居中了

### DISCUSSION THREAD

**Discussion by INSIinc** [tuna/thuthesis#753] [Q&A] [answered]
## 编译环境
编译的系统： Windows 10 
TeX 发行版： MiKTeX 2.9.6753
模板版本：v7.2.4
模板类型： bachelor

## 描述问题
设置了目录标题的段前，段后高度后就没法居中了

截图：
之前
![image](https://user-images.githubusercontent.com/42080694/160291362-6c93d374-ed18-43c6-a260-66d7433854b6.png)

之后
![image](https://user-images.githubusercontent.com/42080694/160291326-a13dc772-d697-410e-b82a-0f9b6f9a8e8f.png)

复现上述问题的代码：
```TeX
\usepackage{tocloft}
\renewcommand{\cfttoctitlefont}{\heiti\zihao{4}}
\renewcommand*\contentsname{\hfill 目\,\qquad\,录 \hfill}
\setlength{\cftbeforetoctitleskip}{6bp}
\setlength{\cftaftertoctitleskip}{0bp}
```

**Comment by zepinglee** [comment, marked as answer]
模板中使用了 `titletoc` 设置目录格式，跟 `tocloft` 不兼容。

**Comment by INSIinc** [comment]
解决了

**Comment by zepinglee** [comment]
@Harry-Chen 感觉没必要“convert to discussion”吧，这个还会把 issue 号+1。
